src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,39 @@
+.card__rating {
+  display: flex;
+  margin-bottom: 24px;
+  height: 16px;
+  width: auto;
+  justify-content: space-between;
+
+  &-reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-text-color;
+  }
+
+  .stars {
+    display: inline-flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    &__star {
+      background-image: url(../images/star.svg);
+      width: 16px;
+      height: 16px;
+      background-position: 50%;
+      background-repeat: no-repeat;
+
+      &:not(:last-child) {
+        margin-right: 4px;
+      }
+    }
+
+    &--4 {
+      .stars__star:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
```

Comment:
reuse all styles from task with stars to make it work for other values like `stars--3`, or `stars--1`